### ISSUE TITLE
Different base model and lora model in prolificdreamer config

### ISSUE THREAD

**Issue by huangyangyi** [threestudio-project/threestudio#99]
Thanks for your impressive re-implementation of prolificdreamer. I noticed that you use `stable-diffusion-2-1-base` for the pre-trained sd model and `stable-diffusion-2-1` to initialize the Lora model, and these two models use schedulers with different prediction types. I don't know if such a setting is from the original prolificdreamer and whether it improves the performance. I would be grateful if you could please provide some explanation about that.

**Comment by bennyguo** [comment]
Thank you! Yes the authors claim that they use v-prediction to train the LoRA model:
<img width="780" alt="image" src="https://github.com/threestudio-project/threestudio/assets/19284678/6878b452-76e7-4957-90e4-8ed6113a8ff1">

In my experiments, v-prediction performs better than epsilon prediction especially after t annealing, where using epsilon prediction often causes over-saturated color. You can also try to compare the two choices yourself :)

**Comment by huangyangyi** [comment]
Exactly, I have tried using epsilon prediction for both models, and it leads to over-saturated color. Thanks for your help!

**Comment by silence401** [comment]
could you explain why do these operator on noise_pred by lora and then calculate loss with pretrain sd noise?
![image](https://github.com/threestudio-project/threestudio/assets/29566012/9d955919-33ef-49eb-8231-880216026494)

**Comment by yuanzhi-zhu** [comment]
Hi @silence401 ,

I think in the v-prediction setting, the output of the lora model should be considered as *v*, and we have to convert it back to the meaning of noise_pred and then calculate loss with pre-train sd noise.
